core/templates/dev/head/css/oppia.css

```diff
@@ -3583,13 +3583,19 @@ md-card.preview-conversation-skin-supplemental-card {
 .oppia-dashboard-container a:hover,
 .oppia-dashboard-container a:active,
 .oppia-dashboard-container a:visited {
-  color: #009688;
+  color: inherit;
 }
 
 .oppia-dashboard-container a:hover {
     text-decoration: underline;
 }
 
+.oppia-dashboard-list-summary {
+  display:inline-block;
+  text-decoration: none !important;
```

Comment:
Can we avoid using !important, perhaps by using a more specific CSS class? We try not to use !important since it's basically an override of the normal CSS priorities.